src/Common/ZooKeeper/ZooKeeperImpl.cpp

```diff
@@ -491,7 +491,7 @@ void ZooKeeper::connect(
                 if (use_compression)
                 {
                     compressed_in.emplace(*in);
-                    compressed_out.emplace(*out, CompressionCodecFactory::instance().get("ZSTD", {}));
+                    compressed_out.emplace(*out, CompressionCodecFactory::instance().get("LZ4", {}));
```

Comment:
actually what do you think about making it a config also like `use_compression` and have `LZ4` be default?

Reply:
so we also exchange that in handshake ?

Reply:
ah yes that part complicates it, let's have just LZ4 for now. 😓